webplugin/js/app/mck-app.js

```diff
@@ -464,6 +464,10 @@ function ApplozicSidebox() {
                 options.hidePostFormSubmit != null
                     ? options.hidePostFormSubmit
                     : widgetSettings && widgetSettings.hidePostFormSubmit;
+            options.disableFormPostSubmit =
+                options.disableFormPostSubmit != null
+                    ? options.disableFormPostSubmit
```

Comment:
`options.disableFormPostSubmit != null` why we are checking null here ?, instead of we can use `options.disableFormPostSubmit ? `